pages/demo/form.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="p-4 space-y-4">
+    <h2 class="text-lg font-bold">Demo: Form</h2>
+    <TForm
+      v-model="data"
+      :fields="fields"
+      :field-config="{ labelPosition: 'top' }"
+      class="space-y-4"
+      submit-label="Submit"
+      @save="submit"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  required,
+  email,
+  minLength,
+  alphaNum,
+  helpers
+} from '@vuelidate/validators'
+import { ref } from '@nuxtjs/composition-api'
+import { useDoc } from '~/use/doc'
+export default {
+  name: 'Form',
+  setup() {
+    const { find, exists } = useDoc('profiles')
+    const data = ref({
+      name: '',
+      email: '',
+      username: '',
+      place: ''
+    })
+    const isUnique = async (value) => {
+      await find('username', value)
+      if (exists.value) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    const ErrorMessages = {
```

Comment:
Can you please dissolve this object and use strings directly in validation declaration?